Tribler/community/market/core/matching_engine.py

```diff
@@ -28,236 +27,226 @@ def __init__(self, order_book):
 
         self.order_book = order_book
 
+    @staticmethod
+    def generate_match_id():
```

Comment:
Would it be a good idea to check if this `match_id` exists in the database before using it? Or do you take care of this in some other way?